doc/whats_new/v1.6.rst

```diff
@@ -35,6 +35,9 @@ Changes impacting many modules
   More details in :ref:`estimator_tags`.
   :pr:`22606` by `Thomas Fan`_ and :pr:`29677` by `Adrin Jalali`_.
 
+- |Enhancement| Scikit-learn classes and functions can be used while only having a
```

Comment:
You will need to add a fragment in `doc/whats_new/upcoming_changes/many-modules/29793.enhancement.rst` and have the following entry:

```
- Scikit-learn classes and functions can be used while only having a
  `import sklearn` import line. For example, `import sklearn; sklearn.svm.SVC()` now works.
  By :user:`Thomas Fan <thomasjpfan>`.
```